web/sites/default/config/user.role.council_clerk.yml

```diff
@@ -35,6 +35,7 @@ permissions:
   - 'create field_date_of_final_disposition'
   - 'create field_disposition_notes'
   - 'create field_efiles_link'
+  - 'create field_finanicial_impact_analysis'
```

Comment:
I merged master into my latest commit, yet this config still appears altered. Do you have a recommendation of what I should try next?

Reply:
Try checking the files that should not have changed out from master. Likely your change is chronologically after the commit that changed master, so it is still going to try to touch the file because of how your branch started. The odd thing is that the master branch technically has these changes already. 🤔 I'm not really sure how you managed that.

Here are the three commits that should not be in your branch:
![2021-02-22 at 2 21 PM](https://user-images.githubusercontent.com/4097978/108777875-5bd12f00-7519-11eb-9bce-88ce07a2cd72.png)

Reply:
@brittanysmart @RichardDavies, I'm not going to get to this as I'm about to slide into another meeting that goes to 5pm. Could one of you take a look at rebasing this branch so we can get a clean merge? I normally would see commits that "merged in master" rather than commits that look like they are from both the current branch owner and the committer of the other branch.

Reply:
@joshuami yep, taking a look now.